bot/exts/info/doc/_cog.py

```diff
@@ -341,10 +341,14 @@ async def get_command(self, ctx: commands.Context, *, symbol_name: Optional[str]
             if doc_embed is None:
                 error_message = await send_denial(ctx, "No documentation found for the requested symbol.")
                 await wait_for_deletion(error_message, (ctx.author.id,), timeout=NOT_FOUND_DELETE_DELAY)
+
```

Comment:
Might as well remove this line, I don't think it helps much in terms of readabily. Good to not introduce unneccessary changes.
```suggestion
```

Reply:
Done :+1: